src/black/__init__.py

```diff
@@ -2709,6 +2709,9 @@ def make_comment(content: str) -> str:
 
     if content[0] == "#":
         content = content[1:]
+    NON_BREAKING_SPACE = " "
+    if content and content[0] == NON_BREAKING_SPACE and "type: ignore" not in content:
```

Comment:
FYI, you can also use Unicode name escapes, so instead of a defining an aptly named variable, you could just replace it with `"\N{NO-BREAK SPACE}"`

Now does it matter? Probably not since it's arguable which one is more readable, but I'd like to point it to you as a language you may have not known :)

Reply:
Thanks I did not know ! 😄